From a9c18a9fc4984ad185b4a6f2536ec0fc908ad2e1 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Mon, 1 Feb 2016 14:32:43 -0500
Subject: [PATCH 0753/1110] drm/amd/dal: Make DCE 10 HWSS independent of DCE 8

Currently Linux kernel tree doesn't have dce 8 code so
we can't reference it from dce 10 hardware sequencer.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc_resource.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc_resource.c b/drivers/gpu/drm/amd/dal/dc/core/dc_resource.c
index d166811..8cb756e 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc_resource.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc_resource.c
@@ -31,8 +31,12 @@
 #include "opp.h"
 #include "transform.h"
 
+#if defined(CONFIG_DRM_AMD_DAL_DCE10_0)
 #include "dce100/dce100_resource.h"
+#endif
+#if defined(CONFIG_DRM_AMD_DAL_DCE11_0)
 #include "dce110/dce110_resource.h"
+#endif
 
 bool dc_construct_resource_pool(struct adapter_service *adapter_serv,
 				struct dc *dc,
-- 
2.7.4

